.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Fixes issue #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
```

Comment:
@jenkoian if there is no issue, does it need to be deleted?

Reply:
Yeah or left blank or something